hazelcast-client/src/main/java/com/hazelcast/client/proxy/ClientMultiMapProxy.java

```diff
@@ -149,6 +150,13 @@ public boolean remove(Object key, Object value) {
         return new UnmodifiableLazyList<V>(resultParameters.response, getSerializationService());
     }
 
+    public void delete(Object key) {
```

Comment:
We should add a test for client-side delete functionality in `com.hazelcast.client.multimap.ClientMultiMapTest`